tests/e2e/spaces_test.go

```diff
@@ -305,6 +307,13 @@ var _ = Describe("Spaces", func() {
 							}},
 						}},
 					})
+
+					app1GUID = getAppGUIDFromName(app1Name)
+					pkgGUID := createBitsPackage(app1GUID)
+					uploadTestApp(pkgGUID, defaultAppBitsFile)
```

Comment:
uploading app code, building it and waiting for the build should not be necessary for this test. I assume that is another workaround for the stste collector failing when there is no droplet

PS: I do not think that this file should be changed as part of this PR at all